openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md

```diff
@@ -4,7 +4,7 @@ In this section, we provide guides and references to use the QuickSight connecto
 
 # Requirements
 <!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/dashboard/quicksight).
+You can find further information on the Quicksight connector in the [docs](https://docs.open-metadata.org/connectors/dashboard/quicksight).
 
```

Comment:
Any more details on the type of permission a user would need? I believe the user's IAM policy should have certain access to extract metadata. Checking the metadata ingestion should give us some indication on the type of operations we perform.

Reply:
makes sense, let me add those permissions requirement here

Reply:
I will resolve the same in https://github.com/open-metadata/OpenMetadata/pull/11179